mlir-sycl/test/Conversion/SYCLToLLVM/sycl-methods-to-llvm.mlir

```diff
@@ -1,121 +1,295 @@
 // RUN: sycl-mlir-opt -split-input-file -convert-sycl-to-llvm="use-bare-ptr-call-conv" %s | FileCheck %s
-// XFAIL: *
 
 //===-------------------------------------------------------------------------------------------------===//
-// sycl.nd_range.get_global_range
```

Comment:
These tests had to be changed because the tests themselves were not written correctly (used old signature of operations). I'll add them with the correct signature in the next patch of this series if that's ok.